companion.php

```diff
@@ -300,24 +320,24 @@ function companion_wp_login() {
 		$urlparts = wp_parse_url( network_site_url() );
 		$domain   = $urlparts['host'];
 		// WP.COM Public.API Endpoint
-		$url      = "https://public-api.wordpress.com/wpcom/v2/jurassic-ninja/extend";
+		$url = "https://public-api.wordpress.com/wpcom/v2/jurassic-ninja/extend";
 		// Retrieve Shared Token from persistent data.
 		$persistent_data = new Atomic_Persistent_Data();
-		$api_token = $persistent_data->JN_API_TOKEN;
+		$api_token       = $persistent_data->JN_API_TOKEN;
 		wp_remote_post( $url, [
 			'headers' => [
 			],
 			'body'    => [
 				'domain'    => $domain,
-                'api_token' => $api_token,
+				'api_token' => $api_token,
 			],
 		] );
 		if ( ! empty ( $auto_login ) ) {
-		    if ( empty( get_option( 'big_sky_api_key' ) ) ) {
-			    wp_safe_redirect( '/wp-admin' );
-		    } else {
-			    wp_safe_redirect( '/wp-admin/site-editor.php' );
-		    }
+			if ( empty( get_option( 'big_sky_api_key' ) ) ) {
+				wp_safe_redirect( '/wp-admin' );
+			} else {
+				wp_safe_redirect( '/wp-admin/site-editor.php' );
+			}
 			exit( 0 );
 		}
 	} else {
```

Comment:
What is this part of the condition for?

Reply:
BigSky is what the AI team is working on and they have been using Jurassic.Ninja to quickly prototype. This gave them a seamless experience of dropping into the editor vs having to do another click.

Reply:
Thanks, it's clear now.